tests/Custom/TimeoutTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Mink\WebdriverClassDriver\Tests\Custom;
+
+use Behat\Mink\Exception\DriverException;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class TimeoutTest extends TestCase
+{
+    /**
+     * @after
+     */
+    protected function resetSessions(): void
+    {
+        $session = $this->getSession();
+
+        // Stop the session instead of only resetting it, as timeouts are not reset (they are configuring the session itself)
+        if ($session->isStarted()) {
+            $session->stop();
+        }
+
+        // Reset the array of timeouts to avoid impacting other tests
+        $session->getDriver()->setTimeouts([]);
+
+        parent::resetSessions();
+    }
+
+    public function testInvalidTimeoutSettingThrowsException(): void
+    {
+        $this->expectException(DriverException::class);
```

Comment:
this should be placed after the `start()` call, to ensure that the exception happens in `setTimouts`

Reply:
Agreed... these tests were copied from MinkSelenium2Driver; they could probably be improved.